models/bamf_nnunet_mr_liver/meta.json

```diff
@@ -0,0 +1,137 @@
+{
+    "id": "",
+    "name": "bamf_nnunet_mr_liver",
+    "title": "AIMI MRI Liver",
+    "summary": {
+        "description": "An nnU-Net based model to segment liver from T1 weighted MRI scans",
+        "inputs": [
+            {
+                "label": "Input Image",
+                "description": "The abdominal T1 weighted  MRI scan of a patient.",
+                "format": "DICOM",
+                "modality": "MR",
+                "bodypartexamined": "LIVER",
```

Comment:
Please change to "Liver"

Reply:
Just curious, what is the reason for deviating from the conventions in the standard regarding capitalization?

It is difficult enough to remember one set of conventions, and now one has to remember two... :-(

Reply:
Uff, yeah we did ([list of values for body part examined](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L) for reference).

Reply:
Please ignore @rahul99. I'll fix this server-side. Sorry for the confusion and thank you Andrey for noticing this!

Reply:
@LennyN95 , I followed this advice on `bamf_nnunet_ct_kidney` PR and updated "KIDNEY" to "Kidney". Should I revert it?

Reply:
No worries @rahul99, I updated the backend such that it is case-insensitive. I might patch all metadata at a later point to the correct all-uppercase spelling but for now it's ok ;)